Index: llvm-toolchain_3.3~svn179457/clang/lib/Basic/Version.cpp
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Basic/Version.cpp	2013-03-26 00:16:38.000000000 +0100
+++ llvm-toolchain_3.3~svn179457/clang/lib/Basic/Version.cpp	2013-04-13 10:29:09.869933183 +0200
@@ -15,6 +15,7 @@
 #include "clang/Basic/LLVM.h"
 #include "llvm/Config/config.h"
 #include "llvm/Support/raw_ostream.h"
+#include "clang/Debian/debian_path.h"
 #include <cstdlib>
 #include <cstring>
 
@@ -121,7 +122,8 @@
 #ifdef CLANG_VENDOR
   OS << CLANG_VENDOR;
 #endif
-  OS << "clang version " CLANG_VERSION_STRING " "
+  OS << "clang version " CLANG_VERSION_STRING "-" DEB_PATCHSETVERSION " "
+
      << getClangFullRepositoryVersion();
 
   // If vendor supplied, include the base LLVM version as well.
