--- a/compiler-rt/make/platform/clang_linux.mk
+++ b/compiler-rt/make/platform/clang_linux.mk
@@ -39,9 +39,9 @@
 test_source = $(ProjSrcRoot)/make/platform/clang_linux_test_input.c
 ifeq ($(CompilerTargetArch),i386)
   SupportedArches := i386
-  ifeq ($(call TryCompile,$(CC),$(test_source),-m64),0)
-    SupportedArches += x86_64
-  endif
+#  ifeq ($(call TryCompile,$(CC),$(test_source),-m64),0)
+#    SupportedArches += x86_64
+#  endif
 else
   SupportedArches := x86_64
   ifeq ($(call TryCompile,$(CC),$(test_source),-m32),0)
