Index: llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.cpp
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Driver/ToolChains.cpp	2013-04-12 00:55:55.000000000 +0200
+++ llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.cpp	2013-04-13 10:29:16.117932941 +0200
@@ -2254,6 +2254,11 @@
     addSystemInclude(DriverArgs, CC1Args, P.str());
   }
 
+  // Under Debian, clang headers are installed into
+  // '/usr/include/clang/VERSION/include/'
+  addSystemInclude(DriverArgs, CC1Args, "/usr/include/clang/" + std::string(CLANG_VERSION_STRING) + "/include/");
+  
+
   if (DriverArgs.hasArg(options::OPT_nostdlibinc))
     return;
 
