Index: llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.h
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Driver/ToolChains.h	2013-04-09 06:35:11.000000000 +0200
+++ llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.h	2013-04-13 10:29:34.213932239 +0200
@@ -125,6 +125,8 @@
   virtual bool isPICDefault() const;
   virtual bool isPIEDefault() const;
   virtual bool isPICDefaultForced() const;
+  virtual void AddClangSystemIncludeArgs(const ArgList &DriverArgs,
+                                        ArgStringList &CC1Args) const;
 
 protected:
   virtual Tool *getTool(Action::ActionClass AC) const;
Index: llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.cpp
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Driver/ToolChains.cpp	2013-04-13 10:29:16.117932941 +0200
+++ llvm-toolchain_3.3~svn179457/clang/lib/Driver/ToolChains.cpp	2013-04-13 10:29:34.217932239 +0200
@@ -952,6 +952,13 @@
   return GoodVersion;
 }
 
+void Generic_GCC::AddClangSystemIncludeArgs(const ArgList &DriverArgs,
+                                      ArgStringList &CC1Args) const {
+  // Under Debian, clang headers are installed into
+  // '/usr/include/clang/VERSION/include/'
+  addSystemInclude(DriverArgs, CC1Args, "/usr/include/clang/" + std::string(CLANG_VERSION_STRING) + "/include/");
+}
+
 /// \brief Less-than for GCCVersion, implementing a Strict Weak Ordering.
 bool Generic_GCC::GCCVersion::operator<(const GCCVersion &RHS) const {
   if (Major != RHS.Major)
