Index: llvm-toolchain_3.3~svn179457/clang/lib/Driver/Tools.cpp
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Driver/Tools.cpp	2013-04-13 10:29:30.445932385 +0200
+++ llvm-toolchain_3.3~svn179457/clang/lib/Driver/Tools.cpp	2013-04-13 10:29:31.945932327 +0200
@@ -33,6 +33,7 @@
 #include "llvm/Support/Host.h"
 #include "llvm/Support/Process.h"
 #include "llvm/Support/raw_ostream.h"
+#include "llvm/Config/config.h"
 
 using namespace clang::driver;
 using namespace clang::driver::tools;
@@ -216,7 +217,7 @@
   // libprofile_rt.so. We used to use the -l:libprofile_rt.a syntax, but that is
   // not supported by old linkers.
   std::string ProfileRT =
-    std::string(TC.getDriver().Dir) + "/../lib/libprofile_rt.a";
+    std::string(TC.getDriver().Dir) + "/../lib/llvm-"+PACKAGE_VERSION+"/lib/libprofile_rt.a";
 
   CmdArgs.push_back(Args.MakeArgString(ProfileRT));
 }
