Index: llvm-toolchain_3.3~svn179457/clang/tools/scan-build/scan-build
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/tools/scan-build/scan-build	2013-03-22 00:14:26.000000000 +0100
+++ llvm-toolchain_3.3~svn179457/clang/tools/scan-build/scan-build	2013-04-13 10:29:44.525931840 +0200
@@ -1522,7 +1522,7 @@
 
 # Find 'clang'
 if (!defined $AnalyzerDiscoveryMethod) {
-  $Clang = Cwd::realpath("$RealBin/bin/clang");
+  $Clang = Cwd::realpath("/usr/bin/clang");
   if (!defined $Clang || ! -x $Clang) {
     $Clang = Cwd::realpath("$RealBin/clang");
   }
