Index: llvm-toolchain_3.3~svn179457/lib/Support/CommandLine.cpp
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/lib/Support/CommandLine.cpp	2013-03-10 00:29:37.000000000 +0100
+++ llvm-toolchain_3.3~svn179457/lib/Support/CommandLine.cpp	2013-04-13 10:29:35.189932201 +0200
@@ -1343,6 +1343,10 @@
     OS << LLVM_VERSION_INFO;
 #endif
     OS << "\n  ";
+#ifdef LLVM_DEBIAN_INFO
+    OS << LLVM_DEBIAN_INFO;
+#endif
+    OS << "\n  ";
 #ifndef __OPTIMIZE__
     OS << "DEBUG build";
 #else
