Index: llvm-toolchain_3.3~svn179457/clang/lib/Lex/Makefile
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/clang/lib/Lex/Makefile	2010-07-18 02:14:47.000000000 +0200
+++ llvm-toolchain_3.3~svn179457/clang/lib/Lex/Makefile	2013-04-13 10:29:49.861931633 +0200
@@ -17,8 +17,10 @@
 LIBRARYNAME := clangLex
 
 ifeq ($(ARCH),PowerPC)
+ifneq ($(shell dpkg-architecture -qDEB_HOST_ARCH),powerpcspe)
 CXX.Flags += -maltivec
 endif
+endif
 
 include $(CLANG_LEVEL)/Makefile
 
