Index: llvm-toolchain_3.3~svn179457/lib/Support/Unix/Memory.inc
===================================================================
--- llvm-toolchain_3.3~svn179457.orig/lib/Support/Unix/Memory.inc	2013-03-14 20:01:00.000000000 +0100
+++ llvm-toolchain_3.3~svn179457/lib/Support/Unix/Memory.inc	2013-04-13 10:29:48.745931676 +0200
@@ -329,7 +329,7 @@
   // FIXME: Can we safely always call this for __GNUC__ everywhere?
   const char *Start = static_cast<const char *>(Addr);
   const char *End = Start + Len;
-  __clear_cache(const_cast<char *>(Start), const_cast<char *>(End));
+  __builtin___clear_cache(const_cast<char *>(Start), const_cast<char *>(End));
 #  elif defined(__mips__)
   const char *Start = static_cast<const char *>(Addr);
 #    if defined(ANDROID)
